packages/boot/test/bootstrapTests/test-liquidation-concurrent-1.ts

```diff
@@ -0,0 +1,557 @@
+      check.vaultNotification(managerIndex, i, {
+        debtSnapshot: {
+          debt: { value: scale6(setups[collateralBrandKey].vaults[i].debt) },
+        },
+        locked: { value: scale6(setups[collateralBrandKey].vaults[i].atom) },
+        vaultState: 'active',
+      });
+    }
+  };
+
+  const placeBid = async (collateralBrandKey: string) => {
+    await buyer.sendOffer(
+      Offers.psm.swap(
+        agoricNamesRemotes,
+        agoricNamesRemotes.instance['psm-IST-USDC_axl'],
+        {
+          offerId: `print-${collateralBrandKey}-ist`,
+          wantMinted: 1_000,
+          pair: ['IST', 'USDC_axl'],
+        },
+      ),
+    );
+
+    const maxBuy = `10000${collateralBrandKey}`;
+
+    for (let i = 0; i < setups[collateralBrandKey].bids.length; i += 1) {
+      const offerId = `${collateralBrandKey}-bid${i + 1}`;
+      // bids are long-lasting offers so we can't wait here for completion
+      await buyer.sendOfferMaker(Offers.auction.Bid, {
+        offerId,
+        ...setups[collateralBrandKey].bids[i],
+        maxBuy,
+      });
+      t.like(readLatest('published.wallet.agoric1buyer'), {
+        status: {
+          id: offerId,
+          result: 'Your bid has been accepted',
+          payouts: undefined,
+        },
+      });
+    }
+  };
+
+  const {
+    collateralBrandKey: collateralBrandKeyA,
+    managerIndex: managerIndexA,
+  } = cases[0];
+  const {
+    collateralBrandKey: collateralBrandKeySt,
+    managerIndex: managerIndexSt,
+  } = cases[1];
+
+  await setupVault(collateralBrandKeyA, managerIndexA);
+  await setupVault(collateralBrandKeySt, managerIndexSt);
+
+  await Promise.all(
+    cases.map(({ collateralBrandKey }) => {
+      placeBid(collateralBrandKey);
+    }),
+  );
+
+  // ---------------
+  //  Change price to trigger liquidation
+  // ---------------
+  console.log('Change prices');
+  await priceFeedDrivers[collateralBrandKeyA].setPrice(
+    setups[collateralBrandKeyA].price.trigger,
+  );
+  await priceFeedDrivers[collateralBrandKeySt].setPrice(
+    setups[collateralBrandKeySt].price.trigger,
+  );
+
+  const liveSchedule = readLatest('published.auction.schedule');
+
+  cases.map(async ({ collateralBrandKey, managerIndex }) => {
+    const metricsPath = metricsPaths[managerIndex];
+
+    // check nothing liquidating yet
+    /** @type {import('@agoric/inter-protocol/src/auction/scheduler.js').ScheduleNotification} */
+    t.is(liveSchedule.activeStartTime, null);
+    t.like(readLatest(metricsPath), {
+      numActiveVaults: setups[collateralBrandKey].vaults.length,
+      numLiquidatingVaults: 0,
+    });
+  });
+
+  // advance time to start an auction
+  console.log('step 1 of 10');
+  await advanceTimeTo(NonNullish(liveSchedule.nextDescendingStepTime));
+
+  cases.map(async ({ collateralBrandKey, managerIndex }) => {
+    const metricsPath = metricsPaths[managerIndex];
+
+    t.like(readLatest(metricsPath), {
```

Comment:
The const reads like it will be used again but isn't.
```suggestion
    t.like(readLatest(metricsPaths[managerIndex]), {
```